app/webpack/observations/identify/components/filters_button.jsx

```diff
@@ -185,6 +177,28 @@ class FiltersButton extends React.Component {
       "CC-BY-NC-SA",
       "CC-BY-NC-ND"
     ];
+
+    const loggedInUser = ( config && config.currentUser ) ? config.currentUser : null;
+    const viewerIsAdmin = loggedInUser && loggedInUser.roles
+      && loggedInUser.roles.indexOf( "admin" ) >= 0;
+    const lastColCheckboxProps = [
+      { param: "sounds", key: "show-filter-sounds", label: I18n.t( "has_sounds" ) },
+      { param: "photos", key: "show-filter-photos", label: I18n.t( "has_photos" ) },
+      {
+        param: "user_id",
+        key: "show-filter-user_id",
+        label: I18n.t( "your_observations" ),
+        checked: CURRENT_USER.id
+      }
+    ];
+    if ( viewerIsAdmin ) {
+      lastColCheckboxProps.splice( 2, 0, {
+        param: "disagreements",
+        key: "show-filter-disagreements",
+        label: I18n.t( "disagreements" ),
+        className: "admin"
```

Comment:
The :before label the admin class is adding isn't displaying quite right for me. The admin label overlaps the checkbox label due to its absolute positioning

Reply:
Yeah, it sort of looked messed up no matter what. At least it makes it really clear it's only meant for testing.